packages/api/src/sequelize/migrations/2024-04-03_00_alter-patient-discovery-result-add-index-patient-id.ts

```diff
@@ -23,9 +23,10 @@ export const up: Migration = async ({ context: queryInterface }) => {
 export const down: Migration = ({ context: queryInterface }) => {
   return queryInterface.sequelize.transaction(async transaction => {
     await queryInterface.removeIndex(tableName, indexToCreate, { transaction });
-    await queryInterface.addIndex(tableName, {
+    await queryInterface.addConstraint(tableName, {
       name: constraintToRemove,
       fields: [constraintToRemoveFieldName],
+      type: "primary key",
       transaction,
     });
```

Comment:
This is a fix for the migration I built last week, I thought I had it fixed but found out I didn't when Sequelize tried to revert a DB migration.

It's not an issue for DB versioning, its just in case we need to revert migrations.